checkov/common/graph/graph_builder/graph_components/blocks.py

```diff
@@ -208,6 +212,9 @@ def _should_add_previous_breadcrumbs(
     ) -> bool:
         return not previous_breadcrumbs or previous_breadcrumbs[-1].vertex_id != change_origin_id
 
+    def extract_additional_changed_attributes(self, attribute_key: str) -> List[str]:
+        return []
```

Comment:
can you add some comment here. to clarify, when to override in the subclass. nothing big 😄